### ISSUE TITLE
Add Pyston support

### ISSUE THREAD

**Issue by kura** [ymyzk/tox-gh-actions#70]
Pyston is part of the python action but it is still installable. It'd be great if you could add a way to detect Pyston and use that.

I tried using using `TOXENV` to make my tests run using Pyston but it kept creating a Python 3.8 virtual environment rather than using my Pyston environment

```yaml
      - name: Set up Pyston
        if: ${{ matrix.python-version == 'pyston' }}
        env:
          TOXENV: pyston,pyston-setproctitle,pyston-uvloop
        run: |
          git clone https://github.com/pyenv/pyenv.git ~/.pyenv
          ~/.pyenv/bin/pyenv install pyston-2.2
          ~/.pyenv/bin/pyenv local pyston-2.2
      - name: unittest
        run: tox -vv
```

This was the tox verbose output

`tox-gh-actions config: {'python': {'3.7': ['py37', 'py37-setproctitle', 'py37-uvloop'], '3.8': ['py38', 'py38-setproctitle', 'py38-uvloop'], '3.9': ['py39', 'py39-setproctitle', 'py39-uvloop'], 'pypy-3.7': ['pypy3', 'pypy3-setproctitle', 'pypy3-uvloop'], 'pyston': ['pyston', 'pyston-setproctitle', 'pyston-uvloop']}, 'env': {}}`

I had to resort to adding an if statement and calling `tox -e env,list` to make it work. I suspect this is due to the scoping of `env` and it only applying to the current step.

**Comment by ymyzk** [comment]
Thanks for proposing an idea. This is implemented in #71.

**Comment by kura** [comment]
Awesome, I will give it a try tonight. Thanks!
